Dockerfile.dapper

```diff
@@ -24,13 +24,17 @@ RUN apt-get update -qq && apt-get install -y --no-install-recommends \
 RUN if [ "$(go env GOARCH)" = "amd64" ]; then \
         curl -sL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b "$(go env GOPATH)/bin" v1.27.0; \
     fi
+## install controller-gen
+RUN if [ "$(go env GOARCH)" = "amd64" ]; then \
+        GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0; \
+    fi
 # -- for make rules
 
 # -- for dapper
 ENV GO111MODULE off
 ENV DAPPER_ENV REPO TAG DRONE_TAG CROSS
 ENV DAPPER_SOURCE /go/src/github.com/rancher/harvester/
-ENV DAPPER_OUTPUT ./bin ./dist
+ENV DAPPER_OUTPUT ./bin ./dist ./deploy
```

Comment:
Can we move `./deploy` to `./dist/deploy`.

Reply:
Why do we need to set `./deploy` output here? I see the crd manifests under this dir in this PR.

Reply:
`./bin` and `./dist` are used to put some generated assets during the CI phase, like `test` report or other intermediate artifacts. However, these generated CRD manifests are not only some artifacts but also the pre-install resources of chart, so I choose to put them under `./deploy/charts/harvester/crds` dir. 

Defined the dapper output here is to make sure we can execute `make generate-manifest`, should not consider this?